README.md

```diff
@@ -4,7 +4,7 @@ A collection of platform-agnostic modules for creating secure data models for cr
 
 ## Modules
 
-This is a monorepo that houses the following packages. Please refer to the READMEs for these packages for installation and usage instructions:
+This repo houses the following packages, which you can examine for installation and usage instructions:
```

Comment:
Thoughts on spelling this out?

```suggestion
This repository houses the following packages, which you can examine for installation and usage instructions:
```

I have been using the full word in public docs.

Reply:
I'd even condense it a bit and add the docs note below:
```suggestion
This repository houses the following packages:
```